dashboards/grafana/README.md

```diff
@@ -0,0 +1,180 @@
+# Grafana Dashboards
+
+This project contains updated Helidon grafana dashboards and the instructions on how to 
+configure and enable additional metrics for Helidon SE and Helidon MP.
+
+> Note: This document and it's contents are work in progress only and not complete. 
+> The information and dashboards may change or be removed.
+
+## Table of Contents
+
+* [Software Pre-requisites](#software-pre-requisites)
+* [References](#references)
+* [Helidon Setup](#helidon-setup)
+  + [Application Identification](#application-identification)
+  + [Required Dependencies](#required-dependencies)
+  + [Enabling Metrics](#enabling-metrics)
+* [Prometheus Setup](#prometheus-setup)
+* [Grafana Setup](#grafana-setup)
+* [grafana Dashboard Images](#grafana-dashboard-images)
+
+## Software Pre-requisites
+
+* JDK21 or JDK24
+* Grafana 10.0.13+
+
+## References
+
+See the following for detailed information in Helidon metrics:
+
+* [SE Metrics](https://helidon.io/docs/latest/se/guides/metrics#basic-and-extended-kpi)
+* [Web Client Metrics](https://helidon.io/docs/latest/se/guides/webclient#WebClient-Metrics#WebClient-Metrics)
+* [MP Metrics](https://helidon.io/docs/latest/mp/metrics/metrics)
+* [MP REST Metrics](https://helidon.io/docs/latest/mp/guides/metrics#controlling-rest-request-metrics)
+* [Helidon Labs Virtual Threads - JDK24](https://github.com/helidon-io/helidon-labs/tree/main/incubator/virtual-threads-metrics)
+
+## Helidon Setup
+
+### Application Identification
+
+To ensure we can identify individual helidon applications as well as allowing drill-through, you should add the
+following labels to your generated metrics using the system property or `application.yaml` (SE) or `microprofile-config.properties` (MP):
+
+**Helidon SE**
+
+* System Property: `-Dmetrics.tags="se_app=my-se-app"`
+* `aplication.yaml`: 
+  ```bash
+  metrics:
+    tags: "se_app=my-se-app"
+  ```
+  
+This will add the label to prometheus generated metrics, e.g.:
+
+```bash
+jvm_uptime_seconds{instance="host.docker.internal:7001", job="helidon", scope="base", se_app="my-se-app"}
+```
+
+**Helidon MP**
+
+* System Property: `-Dmp.metrics.appName=my-mp-app`
+* `microprofile-config.properties` `
+   ```bash
+   mp.metrics.tags=mp_app=my-mp-app
```

Comment:
Similar as above...let's use `appName` in both places:

* System Property: `-Dmp.metrics.appName=my-mp-app`
* `META-INF/microprofile-config.properties`
   ```properties
   mp.metrics.appName=my-mp-app
   ```

Reply:
Its not an issue any more.

Reply:
Right. As long as you use 4.1.7 or later that should be fixed.